docs/metrics/accelerate-wip.md

```diff
@@ -1,9 +1,9 @@
 # Work in progress metrics
 
-Pulse calculates the work in progress metrics based on the status of all pull requests targeting any base branch in your GitHub repositories.
+Pulse calculates the work in progress metrics based on the status of all pull requests targeting any base branch in your repositories.
 
 !!! important
-    You must install the [GitHub integration](../one-click-integrations/github-integration.md) for Pulse to calculate the work in progress metrics.
+    You must install the [GitHub integration](../one-click-integrations/github-integration.md) or the [Bitbucket integration](../one-click-integrations/bitbucket-integration.md) for Pulse to calculate the work in progress metrics.
```

Comment:
@mrfyda I updated this warning to consider the Bitbucket integration (it was only referring to GitHub), but I'm not sure why we have this information here... We don't have it in Changes & reviews metrics, for example. Should we add a similar warning there?

Also, is it really required to install the GitHub integration for Pulse to calculate the WIP metrics? Can't the customer choose to use the Pulse CLI or the API instead?

Reply:
> that's right. the API only ingest data for the Accelerate dashboard. data for the Changes and WIP dashboard is exclusive for the GH and BB integration; and data for the Lead/Cycle time is exclusive from the Jira integration

I confirm this was the reason that led me to mention that the GitHub integration was required for calculating the metrics. :ok_hand: